Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindow.cs

```diff
@@ -214,6 +227,15 @@ private bool HasAssetChanged(SerializedObject serializedAsset)
             return newAssetJson != m_AssetJson;
         }
 
+        private void OnLostFocus()
+        {
+            // Auto-save triggers on focus-lost instead of on every change
+            #if UNITY_INPUT_SYSTEM_INPUT_ACTIONS_EDITOR_AUTO_SAVE_ON_FOCUS_LOST
```

Comment:
I think we should be checking if dirty before calling save here.

Reply:
You're absolutely right, will fix

Reply:
This has now been addressed.